docs/docs/setup-vast/install/docker.md

```diff
@@ -0,0 +1,42 @@
+# Docker
+
+:::tip Docker Hub
+We provide pre-built Docker images at [dockerhub.com/tenzir][dockerhub].
+:::
+
+[dockerhub]: https://hub.docker.com/repository/docker/tenzir/vast
+
+Our Docker image contains a dynamic of VAST build with plugins as shared
+libraries. The system user `vast` runs the VAST executable with limited
+privileges. Database contents go into the volume exposed at `/var/lib/vast`.
+
+## Start the container
+
+Start VAST in a container and detach it to the background.
+
+```bash
+mkdir -p /var/lib/vast
+docker run -dt --name=vast --rm -p 42000:42000 -v /var/lib/vast:/var/lib/vast \
+  tenzir/vast:latest start
```

Comment:
This feels a bit excessive. Isn't there a shorter way?

Reply:
Not really, no.

- `-d` for detaching, i.e., running in background
- `-t` for terminal output
- `--name` to name the image
- `--rm` to clear older volumes
- `-p` to expose the port to the outer world
- `-v` to mount the database directory

Reply:
Thanks, I added this explanation to the instructions.